build/luajit.zig

```diff
@@ -163,9 +163,9 @@ pub fn configure(b: *Build, target: Build.ResolvedTarget, optimize: std.builtin.
 
     lib.linkLibC();
 
-    lib.defineCMacro("LUAJIT_UNWIND_EXTERNAL", null);
+    lib.root_module.addCMacro("LUAJIT_UNWIND_EXTERNAL", "");
```

Comment:
```suggestion
    lib.root_module.addCMacro("LUAJIT_UNWIND_EXTERNAL", "1");
```

I believe this would be the correct migration change; the old [`Step.Compile.defineCMacro`](https://ziglang.org/documentation/0.13.0/std/#std.Build.Step.Compile.defineCMacro) would set the value to `"1"` when `value` is `null`.

Reply:
Seems right.